examples/aggregation.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
```

Comment:
bdbcontrib/examples/ doesn't feel like the right place for this. Consider bdbcontrib/src?